File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/inventory/shared/dtos/substore-requisition.dto.ts
import { SubStoreRequisitionItems_DTO } from "./substore-requisition-item.dto";

export class SubStoreRequisition_DTO {
    CreatedOn: string = null;
    RequisitionNo: number = null;
    IssueNo: number = null;
    DispatchNo: number = null;
    CreatedByName: string = null;
    ReceivedBy: string = null;
    Remarks: string = null;
    RequisitionStatus: string = null;
    IsDirectDispatched: boolean = false;
    verificationId: number = null;
    RequisitionItems: SubStoreRequisitionItems_DTO[];
}

File: Code/Websites/DanpheEMR/Services/WardSupply/Inventory/Requisition/DTOs/SubStoreRequisition_DTO.cs
using System;
using System.Collections.Generic;

namespace DanpheEMR.Services.WardSupply.Inventory.Requisition.DTOs
{
    public class SubStoreRequisition_DTO
    {
        public DateTime CreatedOn { get; set; }
        public int RequisitionNo { get; set; }
        public int? IssueNo { get; set; }
        public int? DispatchNo { get; set; }
        public string CreatedByName { get; set; }
        public string ReceivedBy { get; set; }
        public string Remarks { get; set; }
        public string RequisitionStatus { get; set; }
        public bool IsDirectDispatched { get; set; }
        public int? VerificationId { get; set; }
        public List<SubStoreRequisitionItems_DTO> RequisitionItems { get; set; }
    }
}
